keps/sig-apps/20190318-Graduate-CronJob-to-Stable.md

```diff
@@ -0,0 +1,98 @@
+---
+title: graduate-cronjob-to-stable
+authors:
+  - "@barney-s"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-scheduling
+reviewers:
+  - "@liggitt"
+  - "@kow3ns"
+  - "@janetkuo"
+  - “@mortent”
+approvers:
+  - "@kow3ns"
+  - "@liggitt"
+editor: TBD
+creation-date: 2019-04-18
+last-updated: 2019-04-18
+status: implementable
+see-also:
+  - 
+replaces:
+superseded-by:
+---
+
+# Graduate CronJob to stable
+
+## Table of Contents
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Implementation Notes](#implementation-notes)
+  - [Constraints](#constraints)
+  - [Test Plan](#test-plan)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+
+## Summary
+
+[CronJob](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/) is a Kubernetes API that creates Job object on a schedule specified by a cron spec. It is in beta status for a long time. This document lays out the plan to promote it to stable.
+
+## Motivation
+
+CronJob definition has been stable for the last few releases and is useful to run periodic tasks in kubernetes cluster. This API adds the ability to add cron facility to a cluster. We feel the API is ready to be promoted to Stable and be supported longterm by the community.
+
+### Goals
+
+* Plan to promote CronJob API to stable version.
```

Comment:
I assume this means creating a batch/v1 Cronjob. Might be worth explicitly stating that. 
Would it be worth mentioning here when the batch/v2alpha1 and batch/v1beta1 Cronjob will be deprecated/removed or at least if that they plan to be?

Reply:
It would be useful to call out the API change and that the alpha and beta API versions will follow the deprecation policy at https://kubernetes.io/docs/reference/using-api/deprecation-policy/